persistence/README.md

```diff
@@ -145,6 +145,12 @@ For example, on macOS, you can check for this with `lsof -i:5432` and kill the a
 **Q**: Why not use an ORM?
 **A**: We are trying to keep the module small and lean initially but are ope
```

Comment:
I noticed that the sentence here is truncated :)

Reply:
Hahaha, I think `open to suggestions` is what we wanted to say 😅